migrations/24_deploy_idle_pools.js

```diff
@@ -41,55 +48,34 @@ module.exports = async function (deployer, network, accounts) {
     "0xe4e69ef860d3018b61a25134d60678be8628f780",
   ];
 
-  logTransaction(
-    await poolManager.addPool("0x675eC042325535F6e176638Dd2d4994F645502B9", boosterAdd, 3, 3),
-    "add gauge AATranche_lido"
-  );
-
   await fundLpToken(lp_tokens, lp_tokens_users, admin);
 
+  const gaugeControllerContract = new web3.eth.Contract(gaugeControllerABI, gaugeController);
+  const gaguesCount = toBN(await gaugeControllerContract.methods.n_gauges().call()).toNumber();
```

Comment:
`gagues`, `gaguesCount`

Mistype in naming, doesn't has impact on functionality, but better to rename it